demos/demo_keygen.py

```diff
@@ -96,6 +96,8 @@ def progress(arg=None):
 
     if options.newphrase:
         phrase = getattr(options, 'newphrase')
+        # Turn it into bytes.
+        phrase=phrase.encode("utf-8")
```

Comment:
I think the change to accept unicode passwords in `_write_private_key()` should be in this PR, it's related, and it's not yet too big a PR. But, I'm not authoritative here :)